tests/MediaBrowser.Api.Tests/JellyfinApplicationFactory.cs

```diff
@@ -101,22 +98,21 @@ protected override TestServer CreateServer(IWebHostBuilder builder)
             // Finish initializing the app host
             var appHost = (CoreAppHost)testServer.Services.GetRequiredService<IApplicationHost>();
             appHost.ServiceProvider = testServer.Services;
-            appHost.InitializeServices();
-            appHost.FindParts();
```

Comment:
why is this no longer needed?

Reply:
Those weren't removed, I just updated them to run synchronously: https://github.com/jellyfin/jellyfin/pull/2945/files#diff-01322dfde28c35f44a436fc2ba596206R101